demo/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "demo",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "npm run build && http-server",
+    "build": "webpack --mode production --config ../webpack.demo.config.js"
```

Comment:
After discussion with @nathan-barrett, we decided to create another `webpack.config.js` file but catering to creating a compiled version of dnt-helper inside the demo folder.
Typically, `http-server` package doesn't allow for accessing files outside of the directory it wants to serve (see issue here https://github.com/http-party/http-server/issues/592). So by running another webpack build that sends a compiled file to the `demo` directory, we were then able to access the DNT function and display what we needed in the demo page.